test/system/README.md

```diff
@@ -42,6 +42,13 @@ should be reserved for a first-pass fail-fast subset of tests:
 without having to wait for the entire test suite.
 
 
+Running tests
+=============
+To run the tests locally in your sandbox, you can use one of these methods:
+
+* bats test/system/070-build.bats            # runs just the specified test
```

Comment:
Don't forget the `PODMAN=./bin/podman`! Without that, it uses $PATH

Reply:
added